src/containers/AssignmentTexterContact.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react'
+import $ from 'jquery'
```

Comment:
Can we import it as `jQuery` instead of `$` -- it's really hard to see where it's used.

Reply:
If I'm reading this right, it's only used in one place, `$(window).width()`. I'd like to push to not add jQuery at all, and find a way to do this with native JS, as jQuery is a pretty big package.